types/lodash/common/lang.d.ts

```diff
@@ -1078,7 +1078,7 @@ declare module "../index" {
          * @param value The value to check.
          * @return Returns true if value is NaN, else false.
          */
-        isNaN(value?: any): boolean;
+        isNaN(value?: any): value is number;
```

Comment:
This is in an incorrect use of type guards, because it implies that anything that isn't NaN *isn't* a number.